### DISCUSSION TITLE
Question about GUI

### DISCUSSION THREAD

**Discussion by chaewon0207** [vdemichev/DiaNN#1178] [Q&A]
Hello

I'm a graduate student at Korea university. I'm trying to analysis phosphopeptide by using DIA-NN1.9.1. But i have a problem about GUI

![diann](https://github.com/user-attachments/assets/747ed96e-017d-46fa-ba2f-713c264211c4)

as you can see the picture, missed cleavages and maximum number of variable modification is overlapped so I can not change the missde cleavages and maximum number of  variabl modification.

I changed the resolution of monitor but  I can't fix the problem.
Is there any way to change it?

Thank you

**Comment by vdemichev** [comment]
Hi,

We've seen several reports like this when the system language is set to a language other than English. The DIA-NN GUI is implemented using the old Microsoft WinForms framework (there are other frameworks, but we stick with WinForms because it has a package supporting tabbed-interface, which is super helpful), and how it looks is basically out of our control, it's the Windows scaling mechanisms applying here. So I can only suggest to try switching the system language, or just using 'Additional options' to adjust the settings instead of the GUI controls. 

Best,
Vadim
